libs/editor/src/lib/components/wizard-summarize/wizard-summarize.component.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+
+import { WizardSummarizeComponent } from './wizard-summarize.component'
+import { TranslateModule } from '@ngx-translate/core'
+import { UiModule } from '@lib/ui'
+import { BrowserModule, By } from '@angular/platform-browser'
+import { HttpClientTestingModule } from '@angular/common/http/testing'
+import { NO_ERRORS_SCHEMA } from '@angular/core'
+import { WizardService } from '../../services/wizard.service'
+
+const localStorageMock = () => {
+  let storage = {}
+  return {
+    getItem: (key) => (key in storage ? storage[key] : null),
+    setItem: (key, value) => (storage[key] = value || ''),
+    removeItem: (key) => delete storage[key],
+    clear: () => (storage = {}),
+  }
+}
+
+describe('WizardSummarizeComponent', () => {
+  let component: WizardSummarizeComponent
+  let fixture: ComponentFixture<WizardSummarizeComponent>
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [WizardSummarizeComponent],
+      imports: [
+        TranslateModule.forRoot(),
+        UiModule,
+        BrowserModule,
+        HttpClientTestingModule,
+      ],
+      schemas: [NO_ERRORS_SCHEMA],
+      providers: [WizardService],
+    }).compileComponents()
+  }))
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(WizardSummarizeComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  beforeEach(() => {
+    Object.defineProperty(window, 'localStorage', { value: localStorageMock() })
+
+    window.localStorage.setItem('title', 'title')
+    window.localStorage.setItem('abstract', 'abstract')
+    window.localStorage.setItem(
+      'tags',
+      JSON.stringify([
+        {
+          display: 'tagName1',
+          value: 'tagName1',
+        },
+        {
+          display: 'tagName2',
+          value: 'tagName2',
+        },
+      ])
+    )
+    window.localStorage.setItem('datepicker', String(new Date().getTime()))
+    window.localStorage.setItem('dropdown', JSON.stringify('10000'))
+    window.localStorage.setItem('description', 'description')
+  })
```

Comment:
while you're at dealing with localstorage, you should refactor to be able to handle multiples import => store with the id as key ?

Reply:
storage should be done in a service, implementation being private. If you change implementation then you won't be forced to rewrite all tests.